### PR TITLE
fix(region-info): IAM service principal for China regions

### LINKED ISSUE #2198: incorrect IAM Service Principal for Lambda in China regions
Hello, 

The recent release includes support for IAM service principles for China regions. I did some tests using the CDK example repo. The Classic Load Balancer and Application Load Balancer examples worked without any issue. But when I tried custom-resource example, the deployment failed. Below is the error message: 

Invalid principal in policy: "SERVICE":"lambda.amazonaws.com.cn" 

I found out that the correct IAM service principal for Lambda in China regions is 'lambda.amazonaws.com'。

```
daf7d4f7304ee111e89c2dfa7ae01bbebcServiceRoleFE9ABB04) Invalid principal in policy: "SERVICE":"lambda.amazonaws.com.cn" (Service: AmazonIdentityManagement; Status Code: 400; Error Code: MalformedPolicyDocument; Request ID: 4f413759-59fe-11e9-a3f0-1926a8616aab)
        new Role (/Users/sunhua/Documents/Projects/github.com/aws-samples/aws-cdk-examples/typescript/custom-resource/node_modules/@aws-cdk/aws-iam/lib/role.js:24:22)
```

### PR DESCRIPTION
**Pull request by bnusunny**
Fixes #2198 

----

Please read the [contribution guidelines](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md) and follow the pull-request checklist.

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by bnusunny** [comment]
@RomainMuller Hello, I'm not able to see the CodeBuild details. Can you help to see what failed?

File: packages/@aws-cdk/region-info/build-tools/generate-static-data.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ async function main(): Promise<void> {
     'us-west-1',
     'us-west-2',
     'ap-south-1',
-    // 'ap-east-1',
```
Comment (RomainMuller) [inline packages/@aws-cdk/region-info/build-tools/generate-static-data.ts]: Please leave those commented out - we don't yet have the CDK Metadata resource in those regions.

File: packages/@aws-cdk/region-info/build-tools/generate-static-data.ts [hunk 2]
```diff
@@ -52,6 +52,7 @@ async function main(): Promise<void> {
     'eu-west-2',
     'eu-west-3',
     'eu-north-1',
+    'me-south-1',
```
Comment (RomainMuller) [inline packages/@aws-cdk/region-info/build-tools/generate-static-data.ts]: Please leave those commented out - we don't yet have the CDK Metadata resource in those regions.

File: packages/@aws-cdk/region-info/build-tools/generate-static-data.ts (hunk 1 above)
Comment (bnusunny) [reply packages/@aws-cdk/region-info/build-tools/generate-static-data.ts]: Sure. I comment it out.

File: packages/@aws-cdk/region-info/build-tools/generate-static-data.ts (hunk 2 above)
Comment (bnusunny) [reply packages/@aws-cdk/region-info/build-tools/generate-static-data.ts]: Sure. Done.

**Comment by bnusunny** [comment]
@RomainMuller @eladb @rix0rrr I merged the latest master and updated the new code. The tests are green. Can you please help to review and accept before more changes are merge into master? 

Thanks in advance.

**Pull request merged**